server/templates/place.html

```diff
@@ -17,21 +17,26 @@
 
 {% set main_id = 'dc-places' %}
 {% set page_id = 'page-dc-places' %}
-{% set title = place_name + ' ' + category + ' - ' + _('Place Explorer') %}
+{% if category == '' %}
+  {% set title = place_name + ' - ' + _('Place Explorer') %}
+  {% set description = 'Statistics about economics, health, equity, crime, education, demographics, housing, and environment in {place_name}.'.format(place_name=place_name) %}
```

Comment:
Wrap with `_` for these like described here: https://docs.ckan.org/en/2.9/contributing/string-i18n.html#internationalizating-strings-in-jinja2-templates - then you can drop the `{% if locale == "en" %}` check below.

Also might be worth running through https://github.com/datacommonsorg/website/blob/master/docs/localization.md to ensure the new strings are getting extracted properly (you dont have to commit the new extracted strings to the repo, i think itd be good from a sanity check)

Reply:
Wrapped. I left the locale check because we don't have translations for the descriptions ready yet, otherwise the descriptions for non-english locales will be in english.

I tried the frist step in the localization dogs, but the scripts used now have some errors because they're so old and some of the methods they call are depreciated now. Will look into fixing that after we get this PR in.